dev-python/kaitaistruct/kaitaistruct-0.9.ebuild

```diff
@@ -0,0 +1,18 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{6,7,8,9} )
+
+inherit distutils-r1
+
+DESCRIPTION="Kaitai Struct runtime for Python"
+HOMEPAGE="https://kaitai.io/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
```

Comment:
This line is not needed, *distutils-r1.eclass* will set it for you automatically.

Reply:
Thank you, will remove.